### ISSUE TITLE
[new sidenav] no focus indication for section toggles

### ISSUE THREAD

**Issue by willshowell** [angular/material.angular.io#388]
Currently, there is no visual indication when a section toggle is focused.

cc @amcdnl (great work btw! it looks awesome)

**Comment by amcdnl** [comment]
Done. #389
